@here/harp-mapview/lib/geometry/TileGeometryLoader.ts

```diff
@@ -262,6 +262,32 @@ export class TileGeometryLoader {
         enabledKinds: GeometryKindSet | undefined,
         disabledKinds: GeometryKindSet | undefined
     ) {
+        const decodedTile = this.m_decodedTile;
+
+        // Just a sanity check that satisfies compiler check below.
+        if (decodedTile === undefined) {
+            this.finish();
+            return;
+        }
+
+        this.tile.mapView.taskQueue.add({
```

Comment:
I would rather pass the `TaskQueue` in the `TileGeometryLoader` `contructor` instead of getting it from `MapView`.
That makes the dependeny much clearer.

Reply:
The Tiles could theoretically be from different MapViews, and therefore different TaskQueues

Reply:
wouldn't we have multiple TileGeometryLoader instances in that case?

Reply:
yes, seems like there is one TileGeometryManager using one TileGeometryLoader for for one MapView :) gonna change that